js/stdlib/ts/cfxers.ts

```diff
@@ -282,3 +274,93 @@ export class Wallet {
     throw Error(`Account 'from mnemonic' not supported on Conflux, please use secret key`);
   }
 }
+
+// XXX This is nutty
```

Comment:
I think I need some understanding dropped on me. You're trying to "un-parallelize sends from same sender"... how are there /any/ parallel sends? Every send has an `await` with it... so they are serialized?

Reply:
I noticed this specifically as an issue with accessing the faucet. Something like this:

```
  18   │   const accPlayer_arr = await Promise.all(
  19   │     Array.from({ length: numOfPlayers }, () =>
  20   │       stdlib.newTestAccount(startingBalance)));
```

can cause the faucet to try to send funds to a bunch of people in parallel, and CFX will throw errors about reusing the same nonce.

Reply:
that's from examples/raffle

Reply:
Ah; that makes sense. I think that you might consider a tighter focused attack on only the faucet rather than trying to do this more general thing